gandalf.js

```diff
@@ -1,73 +1,78 @@
 /**
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?